src/sound.c

```diff
@@ -376,7 +376,7 @@ f_sound_playfile(typval_T *argvars, typval_T *rettv)
 {
     long	newid = sound_id + 1;
     size_t	len;
-    char_u	*p, *filepath;
+    char_u	*p, *esc;
```

Comment:
It looks like you reverted some necessary lines.

Reply:
You are right. I've squashed the unicode fix and retest the whole thing. Now it should be right.